discovery-server/src/main/asciidoc/application-config-guide.adoc

```diff
@@ -146,5 +146,10 @@ polaris:
       timeout: 20       # Timeout for rule edit sessions
     etl:
       timeout: 36000    # Timeout for generating snapshots
+  extensions:
+      lnb:
+        - name: Engine Monitoring   # for Engine(Druid) Monitoring
```

Comment:
@minhyun2 I think there should be a description of parent

Reply:
@kyungtaak 
The Engine Monitoring menu is in Beta. Therefore, it was added to the configuration regardless of parent.